tools/auto-fuzz/post_process.py

```diff
@@ -310,11 +311,26 @@ def _merge_runs(trial_dir, successful_runs):
     for run in successful_runs:
         print(os.path.join(trial_dir, run['name']))
 
-        # Copy over the fuzzer
-        src_file = os.path.join(trial_dir, run['name'], "fuzz_1.py")
-        dst_file = os.path.join(next_merged_dir, "fuzz_%d.py" % (idx))
-        idx += 1
-        shutil.copyfile(src_file, dst_file)
+        if language == "python":
+            # Copy over the fuzzer for python project
+            src_file = os.path.join(trial_dir, run['name'], "fuzz_1.py")
+            dst_file = os.path.join(next_merged_dir, "fuzz_%d.py" % (idx))
+            idx += 1
+            shutil.copyfile(src_file, dst_file)
+        elif language == "jvm":
+            # Copy over the fuzzer for java project
+            src_file = os.path.join(trial_dir, run['name'], "Fuzz1.java")
+            dst_file = os.path.join(next_merged_dir, "Fuzz1-%d.java" % (idx))
+
+            # Read in the content of the original Fuzz1.java, changing
+            # the class name to the new one and write the content to
+            # the new destination with new file name.
+            with open(src_file, "r") as fin:
+                with open(dst_file, "w") as fout:
+                    for line in fin:
+                        fout.write(line.replace('/*COUNTER*/', '-%d' % (idx)))
```

Comment:
In [here] (https://github.com/ossf/fuzz-introspector/blob/main/tools/auto-fuzz/base_files.py#L262) for matching the public class with the filename and to avoid same class exists in the same package.